website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -936,7 +936,7 @@ Archived release notes for dbt Cloud from 2023
 
     This maximum limit applies to [multi-tenant instances](/docs/cloud/about-cloud/access-regions-ip-addresses) only, and _does not_ apply to single tenant instances.
 
-    Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2-legacy#section/Pagination) section for more information on this change. 
+    Refer to the [Pagination](https://docs.getdbt.com/dbt-cloud/api-v2#/) section for more information on this change. 
```

Comment:
i might be missing something. i don't see a `Pagination` section on https://docs.getdbt.com/dbt-cloud/api-v2#/

Reply:
You have to scroll down to see it on the page we link to. Can't anchor link to it like we could in the old APIs. I know Gideon is working on enhancements

Reply:
oh i see. i was looking at the sidebar. 

as an idea, maybe "... **Pagination** section on the [API v2](https://docs.getdbt.com/dbt-cloud/api-v2#/) page ..." or something? 

totally non-blocking.